project-base/storefront/components/Blocks/Product/LastVisitedProducts/LastVisitedHelpers.ts

```diff
@@ -0,0 +1,47 @@
+import { getCookie, setCookie } from 'cookies-next';
+import { IncomingMessage, ServerResponse } from 'http';
+
+const LAST_VISITED_MAX_ITEMS = 20;
```

Comment:
We currently display only 4 items. Why do we have 20? Let's save only 4 according to current UI. Projects can easily adjust according to their needs afterward.

Reply:
Fixed.